.github/workflows/release-packages.yaml

```diff
@@ -43,3 +43,42 @@ jobs:
           asset_path: ${{ steps.create_packages.outputs.deb_package }}
           asset_name: ${{ steps.create_packages.outputs.deb_package_name }}
           asset_content_type: application/x-deb
+
+  windows-msi-package:
+  env:
```

Comment:
🚫 Wrong indentation, the properties need to be one indentation level further down!